# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/test/mochitest/test_discardAnimatedImage.html
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 image/test/mochitest/test_discardAnimatedImage.html
--- a/image/test/mochitest/test_discardAnimatedImage.html	Fri Jan 27 19:06:16 2023 +0000
+++ b/image/test/mochitest/test_discardAnimatedImage.html	Fri Jan 27 21:11:42 2023 +0200
@@ -19,7 +19,6 @@
     <img id="infinitepng" src="infinite-apng.png">
     <img id="infinitegif" src="animated1.gif">
     <img id="infinitewebp" src="infinite.webp">
-    <img id="infiniteavif" src="infinite.avif">
     <img id="finitepng" src="restore-previous.png">
     <img id="finitegif" src="animated-gif.gif">
     <img id="finitewebp" src="keep.webp">
@@ -39,12 +38,11 @@
   // Enable discarding for the test.
   SpecialPowers.pushPrefEnv({
     'set':[['image.mem.discardable',true],
-           ['image.webp.enabled',true],
-           ['image.avif.sequence.enabled',true]]
+           ['image.webp.enabled',true]]
   }, runTest);
 }
 
-var gImgs = ['infinitepng', 'infinitegif', 'infinitewebp', 'infiniteavif',
+var gImgs = ['infinitepng', 'infinitegif', 'infinitewebp',
              'finitepng',   'finitegif',   'finitewebp'];
 // If we are currently counting frame updates.
 var gCountingFrameUpdates = false;